decisions/Gondor.txt

```diff
@@ -1789,6 +1789,33 @@ political_decisions = {
 		}
 	}
 	
+	claim_harnendor = {
+		picture = "harnendor_map"
+		potential = {
+			tag = GON
+			HRD = { all_core = { owned_by = GON } }
+			NOT = {
+				has_global_flag = harnendor_claimed_GON
+			}
```

Comment:
We're going to want to tie this into their military recruitment flavor. Gondor could get all of Harondor's cores using normal mechanics, but it wouldn't really make sense if they still had low_military_recruitment -- their expansion flavor is locked behind NOT having that modifier...

`NOT = { has_country_modifier = low_military_recruitment }`